extensions/resteasy-reactive/quarkus-resteasy-reactive-jackson/deployment/src/test/java/io/quarkus/resteasy/reactive/jackson/deployment/test/Person.java

```diff
@@ -17,6 +18,7 @@ public void setFirst(String first) {
         this.first = first;
     }
 
+    @RolesAllowed("admin")
```

Comment:
A minor nuisance with `@RolesAllowed` in this case is that it can't be applied to fields - it can only be applied to methods.